DiscordIan/Model/Geocodio/Result.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Text.Json.Serialization;
+
+namespace DiscordIan.Model.Geocodio
+{
+    public class Result
+    {
+        [JsonPropertyName("address_components")]
```

Comment:
The JSON fetch thing that's in here has an option to automatically map these, the `[JsonPropertyName()]` attribute shouldn't be necessary.